backend/test/routes/projects.test.js

```diff
@@ -39,6 +41,47 @@ testRoutes([
     capabilities: ["projects_read_all"],
     type: routeTypes.General,
   },
+  {
+    request: { method: "GET", url: "/projects/1/lessons-learned" },
+    modelFunction: model.findProjectLessonsLearned,
+    capabilities: ["projects_read_all"],
+    type: routeTypes.General,
+  },
+  {
+    request: { method: "GET", url: "/projects/1/lessons-learned/1" },
+    modelFunction: model.findLessonsLearnedById,
+    capabilities: ["projects_read_all"],
+    type: routeTypes.Specific,
+  },
+  // Todo: Uncomment when validators are added for lessons-learned add/update.
```

Comment:
When you do a npm run lint, does this todo come up, if it doesn't, which i suspect is the case, then we should add the correct comment that ensures this todo shows up, so it makes it easier to find all the todo.  Yes we could to a search for todo.

Reply:
Added an eslint comment for this, will need another approval.